R/xstr_prepare_scenario.R

```diff
@@ -19,15 +19,6 @@
 #'
 #' xstr_prepare_scenario(raw_scenarios)
 xstr_prepare_scenario <- function(scenarios) {
-  # Hack #308
-  if (hasName(scenarios, "weo")) {
-    scenarios$weo <- rename(
-      scenarios$weo,
-      weo_sector = "weo_product",
-      weo_subsector = "weo_flow"
-    )
-  }
```

Comment:
@maurolepore I have updated the `weo` column names in the original datasets. Hence, now there is no need to change the column names inside the code.